packages/open-payments/src/client/grant.ts

```diff
@@ -0,0 +1,33 @@
+import { HttpMethod } from 'openapi'
+import { RouteDeps } from '.'
+import {
+  getASPath,
+  InteractiveGrant,
+  NonInteractiveGrant,
+  GrantRequest
+} from '../types'
+import { post } from './requests'
+
+interface RequestGrantArgs {
+  url: string
+  request: GrantRequest
+}
+
+export interface GrantRoutes {
+  requestGrant(
```

Comment:
```suggestion
  request(
```
or
```suggestion
  requestAccess(
```
https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol#section-2

Reply:
yup, had `request` in my follow-up PR, will do it here